datastructures/BlockProposal.h

```diff
@@ -209,6 +225,19 @@ class BlockProposal : public SendableItem {
         return result;
     }
 
+
+    [[nodiscard]] ptr<vector<ptr<string>>> getSGXAESKeyBatch() const {
+        auto result = std::atomic_load(&sgxAESKeyBatch);
+        return result;
+    }
+
+    void setSGXAESKeyBatch(
+            ptr<vector<ptr<string>>>  _sgxAESKeyBatch) {
+        CHECK_STATE(!sgxAESKeyBatch)
+        sgxAESKeyBatch = _sgxAESKeyBatch;
+    }
+
+
     [[nodiscard]] ptr<std::map<transaction_index, ptr<BiteDataField> > > getBiteDataFields() const {
```

Comment:
do we really need to store BITE data fields? the only place where `getBiteDataFields` is called from only needs transaction indexes and not BITE data itself